app/index.html

```diff
@@ -6,6 +6,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <title>crates.io: Rust Package Registry</title>
+    <meta name="description" content="The Rust community’s crate registry" />
+    <meta name="og:title" content="Rust Package Registry" />
+    <meta name="og:description" content="The Rust community’s crate registry" />
```

Comment:
this feels a bit redundant given that the `<title>` element already contains very similar wording. isn't Facebook falling back to using that anyway?

Reply:
> this feels a bit redundant given that the `<title>` element already contains very similar wording. isn't Facebook falling back to using that anyway?

Yes, but for `og:image` there is no fallback (not sure) and I thought putting other open graph tags also as a best practice.

Reply:
> this feels a bit redundant given that the `<title>` element already contains very similar wording. isn't Facebook falling back to using that anyway?

@t

Reply:
> this feels a bit redundant given that the `<title>` element already contains very similar wording. isn't Facebook falling back to using that anyway?

@Turbo87 

Yes, but for `og:image` there is no fallback (not sure) and I thought putting other open graph tags also as a best practice.

Reply:
Sorry if I made any mistakes :(